### PR TITLE
Refactoring attrholder

### PR DESCRIPTION
**Pull request by UladzimirTrehubenka**

### CODE REVIEW

File: contrib/coredns/policy/attrholder_test.go [hunk 1]
```diff
@@ -29,21 +29,21 @@ func TestActionFromResponse(t *testing.T) {
 		},
 		{
 			resp: &pdp.Response{Effect: pdp.Response_DENY, Obligation: []*pdp.Attribute{
-				{Id: "redirect_to", Value: "10.10.10.10"},
+				{AttrNameRedirectTo, "string", "10.10.10.10"},
```
Comment (rdrozhdzh) [inline contrib/coredns/policy/attrholder_test.go]: Id: AttrNameRedirectTo, Value: "10.10.10.10"

File: contrib/coredns/policy/dnstap/policytap.proto [hunk 2]
```diff
@@ -1,10 +1,11 @@
-syntax = "proto2";
+syntax = "proto3";
```
Comment (rdrozhdzh) [inline contrib/coredns/policy/dnstap/policytap.proto]: please copy the proto from 
https://github.com/Infoblox-CTO/golang-dnstap/blob/Coredns-Support/dnstap.pb/policytap.proto
to make sure they are the same

File: contrib/coredns/policy/attrholder_test.go (hunk 1 above)
Comment (UladzimirTrehubenka) [reply contrib/coredns/policy/attrholder_test.go]: Fixed

File: contrib/coredns/policy/dnstap/policytap.proto (hunk 2 above)
Comment (UladzimirTrehubenka) [reply contrib/coredns/policy/dnstap/policytap.proto]: Fixed

**Pull request merged**